README.md

```diff
@@ -1,2 +1,135 @@
+There are 2 primary voice-specific files in this voice build. They are, relative to the voice directory: 
+- `etc/txt.done.data`, which contains normalized prompts with inserted phrasing tags
+- `festvox/lexicon.scm` which contains a mix of manual and automatic g2p transcriptions of all words that occur in the source data.
+
+For the initial voice, [the Grammatek TTS frontend](https://github.com/grammatek/tts-frontend) was used with phrasing enabled, but as of yet this is a very time consuming step. The phrasing step can optionally be skipped to greatly speed things up, in which case pause tags are only inserted where punctuation is present in the prompts.
+
+Due to serious limitations of Flite in terms of handling phoneme names, we use a modified version of SAMPA, which does not use underscores, digits or colons.
+
+<details>
+<summary>See mapping from standard SAMPA to our modified version</summary>
+
+|SAMPA|flite phoneset|
+|-----|--------------|
+|a:   |aa            |
+|a    |a             |
+|ai:  |aii           |
+|ai   |ai            |
+|au:  |auu           |
+|au   |au            |
+|c    |c             |
+|C    |C             |
+|c_h  |ch            |
+|D    |D             |
+|ei:  |eii           |
+|ei   |ei            |
+|E:   |EE            |
+|E    |E             |
+|f    |f             |
+|G    |G             |
+|h    |h             |
+|i:   |ii            |
+|i    |i             |
+|I:   |II            |
+|I    |I             |
+|j    |j             |
+|k_h  |kh            |
+|k    |k             |
+|l    |l             |
+|l_0  |lz            |
+|m    |m             |
+|m_0  |mz            |
+|n    |n             |
+|n_0  |nz            |
+|J    |J             |
+|J_0  |Jz            |
+|N    |N             |
+|N_0  |Nz            |
+|9:   |oee           |
+|9    |oe            |
+|9i:  |oeii          |
+|9i   |oei           |
+|ou:  |ouu           |
+|ou   |ou            |
+|O:   |OO            |
+|Oi   |Oi            |
+|O    |O             |
+|p_h  |ph            |
+|p    |p             |
+|r    |r             |
+|r_0  |rz            |
+|s    |s             |
+|t_h  |th            |
+|t    |t             |
+|u:   |uu            |
+|u    |u             |
+|v    |v             |
+|x    |x             |
+|Y:   |YY            |
+|Yi   |Yi            |
+|Y    |Y             |
+|T    |T             |
+
+</details>
+
+When generating `<voice_dir>/festvox/lexicon.scm` this mapping needs to be (as of yet, manually) applied to the outputs after the G2P step is run.
+
+### Dependencies
+The voice build relies on a few core toolkits, which each have their own installation instructions.
+- [Edinburgh Speech Tools](https://github.com/festvox/speech_tools)
+- [Festvox](https://github.com/festvox/festvox)
+- [Festival](https://github.com/festvox/festival)
+- And finally, [a modified version of CMU Flite](https://github.com/grammatek/Flite)
```

Comment:
Could you maybe add somewhere a description of the icelandic voice addition in the modified version of the Flite repository as it's missing there ?